app/models/form.rb

```diff
@@ -688,6 +688,12 @@ def organization_has_form_approval_enabled?
 
   # use this validator to provide soft UI guidance, rather than strong model validation
   def ensure_a11_v2_format
+    # ensure `answer_01` is a big thumbs question
+    question_1 = self.ordered_questions.find { |q| q.answer_field == "answer_01" }
+    if question_1.question_type != 'big_thumbs_up_down_buttons'
```

Comment:
adds a validation error that if the a11 form isn't the big thumb up/down (which is what the A-11 v2 Form Template comes with). this lets the user know if their form may be out of alignment with standard reporting.

this validation is only run in a view. It does not prevent a record from being saved; but does display on the screen.

![Screenshot 2024-09-09 at 8 53 31 PM](https://github.com/user-attachments/assets/3e6a481b-238a-4b9e-9291-7ebced48a8cd)